ghc-9.2.5.stack.yaml

```diff
@@ -0,0 +1,19 @@
+resolver: lts-20.3
```

Comment:
Please remove the stack.yaml and stack.yaml.lock files. These are anyway not shipped with the distribution.

Reply:
Done. I was able to test `ghc-9.2.5` with `tack test --resolver=lts-20.3`.